tests/e2e/vsan_stretched_cluster.go

```diff
@@ -2637,23 +2837,25 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		err = fpod.WaitForPodsRunningReady(client, csiNs, int32(csipods.Size()), 0, pollTimeout, nil)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-		ginkgo.By("Verifying statefulset scale up/down went fine on sts1 and sts2")
-		// Scale up replicas of statefulset1 and verify CNS entries for volumes
-		scaleUpStsAndVerifyPodMetadata(ctx, client, namespace, statefulset1,
-			replicas1, false, true)
-		// Scale down replicas of statefulset2 and verify CNS entries for volumes
-		scaleDownStsAndVerifyPodMetadata(ctx, client, namespace, statefulset2,
-			ss2PodsBeforeScaleDown, replicas2, false, true)
+		if !rwxAccessMode {
+			ginkgo.By("Verifying statefulset scale up/down went fine on sts1 and sts2")
```

Comment:
Can you give a reason as to why are we removing this for RWX ?

Reply:
updated the code